5.0/fr/0x10-V1-Encoding-and-Sanitization.md

```diff
@@ -0,0 +1,103 @@
+# V1 Encodage et nettoyage
+
+## Objectif du contrôle
+
+Ce chapitre aborde les failles de sécurité les plus courantes des applications web liées au traitement non sécurisé de données non fiables. Ces failles peuvent entraîner diverses vulnérabilités techniques, où les données non fiables sont interprétées selon les règles syntaxiques de l'interpréteur concerné.
+
+Pour les applications web modernes, il est toujours préférable d'utiliser des API plus sûres, telles que les requêtes paramétrées, l'échappement automatique ou les Framework de création de modèles. Dans le cas contraire, un encodage, un échappement ou une purification de sortie soigneusement effectués deviennent essentiels à la sécurité de l'application.
+
+La validation des entrées constitue un mécanisme de défense en profondeur contre les contenus inattendus ou dangereux. Cependant, son objectif principal étant de garantir que le contenu entrant répond aux attentes fonctionnelles et métier, les exigences y afférentes sont décrites dans le chapitre « Validation et logique métier ».
```

Comment:
"les exigences y afférentes" → Is this correct? Isn't it "les exigences afférentes"?